bionic/flow.py

```diff
@@ -844,46 +907,60 @@ def to_builder(self):
 
         return FlowBuilder._from_state(self._state)
 
-    def get(self, name, fmt=None):
+    def get(self, name, collection=None, object_type=object):
         """
         Computes the value(s) associated with an entity.
 
-        If the entity has multiple values, the ``fmt`` parameter indicates
-        how to handle them.  It can have any of the following values:
+        If the entity has multiple values, the ``collection`` parameter
+        indicates how to handle them.  It can have any of the following values:
 
-        * ``object``: return a single value or throw an exception
+        * ``None``: return a single value or throw an exception
         * ``list`` or ``'list'``: return a list of values
         * ``set`` or ``'set'``: return a set of values
         * ``pandas.Series`` or ``'series'``: return a series whose index is
           the root cases distinguishing the different values
 
+        The user can specify the type of object to return in the collection
+        e.g. object (a value in-memory) or BlessedPath (a path to the
+        persisted file for the computed entity)
+
         Parameters
         ----------
 
         name: String
             The name of a entity.
-        fmt: String or type, optional, default is ``object``
+
+        collection: String or type, optional, default is ``None``
             The data structure to use if the entity has multiple values.
 
+        object_type: String
+            The type of object to return in the collection.
+
         Returns
         -------
 
         The value of the entity, or a collection containing its values.
         """
 
         result_group = self._deriver.derive(name)
-        if fmt is None or fmt is object:
-            if len(result_group) == 0:
+        if object_type is object:
+            results = [result.value for result in result_group]
```

Comment:
nit: Let's use a different name than `results` for the new values, since they're no longer `Result` objects.

Reply:
`values` works for me!